mmv1/templates/terraform/examples/backup_dr_backup_vault_full.tf.tmpl

```diff
@@ -13,6 +13,7 @@ resource "google_backup_dr_backup_vault" "{{$.PrimaryResourceId}}" {
     annotations2 = "baz1"
   }
   force_update = "true"
-  force_delete = "true"
```

Comment:
Is the new field supposed to replace `force_delete`? It's not entirely clear. Typically if the new field is meant to replace an existing field we would need to wait for a major release in order to get this merged since this is considered a breaking change for users.

Reply:
field 'ignore_inactive_datasources' is supposed to replace force_delete; field 'ignore_backup_plan_references' is corresponds to the new parameter added to the delete url. When will the next major release be scheduled? My team is hoping to conclude everything by November 10th. Thank you!

Reply:
Hi Mauricio @BBBmau, just wanted to follow up. Please let me if there's more I can learn about the release schedule, thanks!

Reply:
We schedule major releases yearly. We recently had 6.0.0 so the next major release would be sometime next August.

Reply:
We are in google-beta and our product has not been public yet, we are going public early November. Will this still be considered as a breaking-change? We strongly prefer not having to wait till 2025 to release this change.

Reply:
One thing to wonder is will both fields be supported by the API? What's preventing the ability to support both fields? It should be possible to include both by including `pre_delete` and based on the field update the `delete_url` appropriately. An example of this can be found here: 

https://github.com/GoogleCloudPlatform/magic-modules/blob/d0b27ed1a80d3d3449f0cd22f2afa3a784b8cbaa/mmv1/templates/terraform/pre_create/vertex_ai_models.tmpl

Reply:
we added a new field ignoreBackupPlanReferences in our api delete request, but the proto field name in the delete request url stays as force. Consequently, we added a new field ignore_backup_plan_references to Terraform. We wanted to change the parameter name in terraform from force-delete to ignore_inactive_datasources to differentiate the two delete scenarios. I've added a pre_delete function, and left the original force-delete field as it besides marking it deprecated and making it mutually exclusive with  ignore_inactive_datasources.